x-pack/plugins/security_solution/public/management/cypress/e2e/response_actions/isolate_mocked_data.cy.ts

```diff
@@ -92,9 +92,6 @@ describe('Isolate command', { tags: ['@ess', '@serverless', '@brokenInServerless
   describe.skip(
```

Comment:
just fyi, created ticket for our team to re-enable this: https://github.com/elastic/security-team/issues/9626